src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -420,6 +420,16 @@ public RemoteBuildEventUploadModeConverter() {
               + " --experimental_remote_cache_compression_threshold.")
   public boolean cacheCompression;
 
+  @Option(
+      name = "remote_use_output_paths",
+      defaultValue = "true",
```

Comment:
Can you open a separate PR for the flag flip commit? When the PR is imported, all the individual commits are merged into one.

Reply:
will do

Reply:
https://github.com/bazelbuild/bazel/pull/23582